File: HackerRank Solutions/Problem Solving/Mathematics/Fundamentals/Minimum Height Triangle/Minimum Height Triangle.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.lang.*;
import java.util.regex.*;

public class Solution {

    static int lowestTriangle(int base, int area){
        // Complete this function
        return (int)(Math.ceil((2 * (double)area) / (double)base));
    }

    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int base = in.nextInt();
        int area = in.nextInt();
        int height = lowestTriangle(base, area);
        System.out.println(height);
    }
}

File: HackerRank Solutions/Problem Solving/Mathematics/Fundamentals/Minimum Height Triangle/Minimum Height Triangle.cpp
#include <bits/stdc++.h>

using namespace std;

int lowestTriangle(int base, int area){
    // Complete this function
    return(ceil((float)(2 * area)/base));
}

int main() {
    int base;
    int area;
    cin >> base >> area;
    int height = lowestTriangle(base, area);
    cout << height << endl;
    return 0;
}